samples/subsys/logging/syst/src/main.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include <sys/printk.h>
+
+#define DATA_MAX_DLEN 8
+#define LOG_MODULE_NAME syst
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_DBG);
+
+struct test_frame {
+	u32_t id_type : 1;
+	u32_t rtr     : 1;
+	/* Message identifier */
+	union {
+		u32_t std_id  : 11;
+		u32_t ext_id  : 29;
+	};
+	/* The length of the message (max. 8) in byte */
+	u8_t dlc;
+	/* The message data */
+	union {
+		u8_t data[8];
+		u32_t data_32[2];
+	};
+} __packed;
+
+void main(void)
+{
+	struct test_frame frame = { 0 };
+	const u8_t data[DATA_MAX_DLEN] = { 0x01, 0x02, 0x03, 0x04,
+					0x05, 0x06, 0x07, 0x08 };
+	struct log_msg_ids src_level = {
+		.level = LOG_LEVEL_INTERNAL_RAW_STRING,
+		.source_id = 0, /* not used as level indicates raw string. */
+		.domain_id = 0, /* not used as level indicates raw string. */
+	};
+
+
+	/* standard print */
+	LOG_ERR("Error message example.");
+	LOG_WRN("Warning message example.");
+	LOG_INF("Info message example.");
+	LOG_DBG("Debug message example.");
+
+	for (int i = 0; i <= 4; i++) {
```

Comment:
what's the purpose of this for loop? why not simply put 5 log messages one by one?

Reply:
this confuses me too

Reply:
my fault